src/styles/index.css

```
@@ -0,0 +1,36 @@
+@import url(reset.css);
```

Stylelint (17.9.0)

[error] Expected "url(reset.css)" to be ""reset.css"" (import-notation)

(import-notation)